packages/ember-htmlbars/tests/helpers/each_test.js

```diff
@@ -276,6 +276,35 @@ QUnit.test("View should not use keyword incorrectly - Issue #1315", function() {
   equal(view.$().text(), 'X-1:One 2:Two Y-1:One 2:Two ');
 });
 
+Ember.runInDebug(function() {
```

Comment:
Oh, thank you... but it looks like the tests passed (on Easter Sunday, no less, perhaps I'm on Jesus's good side for once). But should I change out of Ember.runInDebug anyway?